### DISCUSSION TITLE
Subject (from XMP category) Search

### DISCUSSION THREAD

**Discussion by psyciknz** [photoprism/photoprism#4381] [Q&A]
I'm trying to work out the search filter I need to Search for one or more of these strings in the subject.  They are populated from XMP categories via XNViewMP

![image](https://github.com/photoprism/photoprism/assets/5350515/e35ec71f-2cd6-45ca-8607-d0d49e3bc233)

I've made the following structure in XNView And are using the XMP files for an offline album metadata store.   So that when indexes in Photoprism, if I have to restore the entire system, I can just search for the album name (Around Santorini) - and make a new album of the results.
![image](https://github.com/photoprism/photoprism/assets/5350515/d60e4615-f2a0-490b-939a-ac41b66cdc16)

The trouble is, I cannot work out the search string I need.   There are about 53 photos tagged with that full string, or at least "Albums, 2019 World Trip, Around Santorini" - being the 3 categorires.

I tried
- subject: key word with variances of the entire subject line, Around+Santorini "Around Santorini" - all gave 0 results
- just "Around Santorini" with no subject gave me pages and pages of results, I suspect found the Santorini as part of the location
- subjects rather than subject

What else can I try?  Sort of hacking around in the DB and generating an album that way, it seems strange that something in the subject fields can't easily be used as a search metadata

**Comment by graciousgrey** [comment]
Unfortunately there is no dedicated search filter available yet for the subject from the photo details. This is planned for the future.
The search filter "subject" is an alias for person and is looking for matches in people that have been tagged on a photo. 

- https://docs.photoprism.app/user-guide/search/filters/#filter-reference

Until we had time to add a search filter for the subject from the photo details you could search for `World & Trip & Santorini` or `Around & Santorini`. If your files do have the date in their metadata you could also add year:2019 `Around & Santorini year:2019`

**Reply by psyciknz** [reply]
Hi

thanks for that.  Seems workable. I tried with all the strings and &
